contrib/vagrant/scripts/helpers.bash

```diff
@@ -54,27 +54,18 @@ esac
 
 kubernetes_master="${controllers_ips[0]}"
 
-# Default values for IPv4
+# Default values for IPv4 and IPv6
 #
 # CIDR Range for Pods in cluster.
-k8s_cluster_cidr=${K8S_CLUSTER_CIDR:-"10.16.0.0/12"} # 10.16.0.1-10.31.255.255
+k8s_cluster_cidr=${K8S_CLUSTER_CIDR:-"10.16.0.0/12,FD02::/96"} # 10.16.0.1-10.31.255.255
 # Mask size for node cidr in cluster.
-k8s_node_cidr_mask_size=${K8S_NODE_CIDR_MASK_SIZE:-"16"} # 1st Node: 10.16.0.1-10.16.255.254, 2nd Node: 10.17.0.1-10.17.255.254...
+k8s_node_cidr_v4_mask_size=${K8S_NODE_CIDR_MASK_SIZE:-"16"} # 1st Node: 10.16.0.1-10.16.255.254, 2nd Node: 10.17.0.1-10.17.255.254...
+k8s_node_cidr_v6_mask_size=${K8S_NODE_CIDR_MASK_SIZE:-"112"} # 1st Node: 10.16.0.1-10.16.255.254, 2nd Node: 10.17.0.1-10.17.255.254...
 # CIDR Range for Services in cluster.
-k8s_service_cluster_ip_range=${K8S_SERVICE_CLUSTER_IP_RANGE:-"172.20.0.0/24"}
-cluster_dns_ip=${K8S_CLUSTER_DNS_IP:-"172.20.0.10"}
-cluster_api_server_ip=${K8S_CLUSTER_API_SERVER_IP:-"172.20.0.1"}
-
-# Default values for IPv6
-#
-# CIDR Range for Pods in cluster.
-#k8s_cluster_cidr=${K8S_CLUSTER_CIDR:-"FD02::/96"} # 10.0.0.1-10.63.255.254
-# Mask size for node cidr in cluster.
-#k8s_node_cidr_mask_size=${K8S_NODE_CIDR_MASK_SIZE:-"112"} # 1st Node: 10.0.0.1-10.0.255.254, 2nd Node: 10.1.0.1-10.1.255.254...
-# CIDR Range for Services in cluster.
-#k8s_service_cluster_ip_range=${K8S_SERVICE_CLUSTER_IP_RANGE:-"FD03::/112"}
-#cluster_dns_ip=${K8S_CLUSTER_DNS_IP:-"FD03::A"}
-#cluster_api_server_ip=${K8S_CLUSTER_API_SERVER_IP:-"FD03::1"}
+k8s_service_cluster_ip_range=${K8S_SERVICE_CLUSTER_IP_RANGE:-"172.20.0.0/24,FD03::/112"}
+cluster_dns_ip=${K8S_CLUSTER_DNS_IP:-"172.20.0.10,FD03::A"}
+cluster_api_server_ipv4=${K8S_CLUSTER_API_SERVER_IPV6:-"172.20.0.1"}
+cluster_api_server_ipv6=${K8S_CLUSTER_API_SERVER_IPV4:-"FD03::1"}
```

Comment:
The naming "K8S_CLUSTER_API_SERVER_IPV4" and "K8S_CLUSTER_API_SERVER_IPV6" doesn't seem to align and it looks like the addresses got exchanged.

Reply:
oh good point! :facepalm: